tap_jira/context.py

```diff
@@ -54,7 +54,11 @@ def bookmark(cls, paths):
     def set_bookmark(cls, path, val):
         if isinstance(val, datetime):
             val = utils.strftime(val)
-        cls.bookmark(path[:-1])[path[-1]] = val
+
+        if val is None:
```

Comment:
Removing bookmarks wasn't possible, it would previously just get set to `null`.  This actually removes the value.